uavf_2024/imaging/target_tracker.py

```diff
@@ -33,8 +33,31 @@ def closest_match(self, target_desc: TargetDescription) -> Target3D:
         '''
         best_match, best_score = self._targets[0], 0
 
+        
+        '''Converts the target description value from an int into a numpy prob distribution array to be dot producted with the list of targets 
+            '''
+        target_desc_list = list(vars(target_desc).values())
+
+        target_desc = TargetDescription(
+                    np.eye(13)[target_desc_list[0]],
+                    np.eye(35)[target_desc_list[1]],
+                    np.eye(8)[target_desc_list[2]],
+                    np.eye(8)[target_desc_list[3]],
+                )
+        
+        for target_detail, target_probability in vars(target_desc).items():
+                target_probability[ target_probability == 0 ] = 10 ** -4 
```

Comment:
Not _super_ important but this makes each of the arrays not a probability distribution anymore because the elements won't sum to 1. The approach I was thinking of is just adding the small number to all the elements, then dividing the entire array by its sum. That way you avoid loops.

Also protip: you can write `10**-4` as `1e-4` as a shortcut.